roles/confluent.ssl/tasks/create_keystore_and_truststore.yml

```diff
@@ -27,11 +27,11 @@
 
 - name: Create Keystore and Truststore with Custom Certs
   include_tasks: custom_certs.yml
-  when: ssl_custom_certs|bool
+  when: ssl_custom_certs|bool or kafka_connect_replicator_ssl_enabled|bool or kafka_connect_replicator_consumer_ssl_enabled|bool or kafka_connect_replicator_producer_ssl_enabled|bool or kafka_connect_replicator_monitoring_interceptor_ssl_enabled|bool
```

Comment:
i would try and move the component specific logic out of the ssl role. it is not meant to know anything about the components and only handle creating keystores/truststores